capeinfra/swimlanes/private.py

```diff
@@ -26,7 +26,10 @@ def __init__(self, name, opts=None):
                 enable_dns_support=True,
                 # NOTE: to set the name of a VPC resource, you have to add a
                 #       tag with the key `Name`. yay consistency :smh:
-                tags={"Name": f"{name}-vpc"},
+                tags={
+                    "Name": f"{name}-vpc",
```

Comment:
Why does this one have a name and none of the others?

Reply:
I assume you're asking about the `Name` tag (rather than `{name}` in the f-string). If so, see the comment above the `tags` definition. I changed the comment a tad to hopefully be clearer.